utility/utils.py

```diff
@@ -870,3 +871,15 @@ def update_dict_recursively(d, u):
         else:
             d = {k: u[k]}
     return d
+
+
+def get_random_str(size=13):
+    """
+    generates the random string of 13 characters
+
```

Comment:
This function doesn't return just random 13 chars but whatever size you define, so please change the docstring accordingly. I won't block this PR and merge it now, but plese fix in some next PR @vavuthu 
```python
args:
    size (int)
```

Reply:
Done